### PR TITLE
[BER-2017] Add information about Andrea Giardini

### PR DESCRIPTION
**Pull request by range**
Add bio and picture for Andrea.

### CODE REVIEW

**Comment by kmugrage** [comment]
@range - can you fix the build error please?

5:07:38 AM: ERROR 2017/09/19 09:07:38 frontmatter delimiter must be followed by CR+LF or LF but those can't be found at filepos 105
5:07:38 AM: Error: Error building site: Errors reading pages: Error: frontmatter delimiter must be followed by CR+LF or LF but those can't be found at filepos 105 for andrea-giardini.md

File: content/events/2017-berlin/speakers/andrea-giardini.md [hunk 1]
```diff
@@ -5,4 +5,10 @@ image = ""
 type = "speaker"
 linktitle = "andrea-giardini"
 
-+++
```
Comment (phrawzty) [inline content/events/2017-berlin/speakers/andrea-giardini.md]: This needs to be `+++` on a single line or else you'll get this error (currently):
```
Error: Error building site: Errors reading pages: Error: frontmatter delimiter must be followed by CR+LF or LF but those can't be found at filepos 105 for andrea-giardini.md
```
Please add a newline after the `+++` and begin your profile information on the next line.

Comment (range) [reply content/events/2017-berlin/speakers/andrea-giardini.md]: Yeah, vim (ha! blame it on the editor) kind of mangled that line. I still need to find out why I don't get notifications via mail anymore.

File: content/events/2017-berlin/speakers/andrea-giardini.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
 +++
 Title = "Andrea Giardini"
 Twitter = ""
-image = ""
+image = "andrea-giardini.jpg"
```
Comment (kmugrage) [inline content/events/2017-berlin/speakers/andrea-giardini.md]: I added the image name here so the image appears on the site

**Pull request merged**